src/components/Reviews/Review.js

```diff
@@ -63,6 +65,22 @@ class Review extends Component {
         ))
     }
   }
+
+  platform(review) {
+    if (review.platform && review.platform.name && review.platform.url) {
+      return <span> Review Origin: <Link to={review.platform.url}>{review.platform.name} </Link> </span>
```

Comment:
`<Link to>` is for linking inside the single page app, for outside links you need `<a href>`

You can see that the url to Upwork that shows up does not work